### ISSUE TITLE
Backspace key deletes unintended object

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/circuit-construction-kit-common#1292]
**Device**
Lenovo
**OS**
Win 11
**Browser**
Chrome
**Problem Description**
For https://github.com/phetsims/qa/issues/1365
If you select an object with keyboard nav, then tab to a different object but don't select it, and press backspace, the subjected object is deleted. This is mostly a problem if trying to delete the voltmeter or contact ammeter, as you can't select them by pressing space on their main body. This isn't necessarily a bug, but feedback on something I encountered.

**Comment by arouinfar** [comment]
> Though Case B still happens if you try to press delete when focusing on a voltmeter or ammeter probe, rather than the main body. If acceptable, please close.

Ah, good catch. Pressing delete on a focused probe should do precisely nothing. Delete to return to toolbox should work for focused circuit components and tool bodies (but not probes).

**Comment by arouinfar** [comment]
> We cannot use the rule "only delete the item if it has keyboard focus" because we must be able to select an item with pointer (touch/mouse) and press delete to delete it. 

Oh, good point. I'd forgotten about the pointer interaction aspect. 

>  @arouinfar what do you recommend?

In this case, I'd leave things as-is. It's a bit of an edge case, anyway. Closing.
